src/trex/utils.py

```diff
@@ -13,3 +18,42 @@ def format(self, record):
         if record.levelno != logging.INFO:
             record.msg = "{}: {}".format(record.levelname, record.msg)
         return super().format(record)
+
+
+def molecule_list_2_df(mol_list: List[Molecule]) -> pd.DataFrame:
+    """Cast list of Molecules to pandas DataFrame"""
```

Comment:
I’d prefer to not use the term "cast", but "convert" instead. "Casting" is IMO not used as a term so much in Python.